Documentation/subcommands/run.md

```diff
@@ -84,16 +84,81 @@ rkt: warning: signature verification has been disabled
 
 ## Mount Volumes into a Pod
 
-Volumes are defined in each ACI and are referenced by name. Volumes can be exposed from the host into the pod (`host`) or initialized as empty storage to be accessed locally within the pod (`empty` pending [rkt #378][rkt #378]). Each volume can be selectively mounted into each application at differing mount points or not mounted into specific apps at all.
+Each ACI can define a [list of mount points](https://github.com/appc/spec/blob/master/spec/aci.md#image-manifest-schema) that the app is expecting external data to be mounted into:
 
-[rkt #378]: https://github.com/coreos/rkt/issues/378
+```json
+{
+    "acKind": "ImageManifest",
+    "name": "example.com/app1",
+    ...
+    "app": {
+        ...
+        "mountPoints": [
+            {
+                "name": "data",
+                "path": "/var/data",
+                "readOnly": false
+            }
+        ]
+    }
+    ...
+}
+```
+
+To fulfill these mount points, volumes are used. A volume is asigned to a mount point if they both have the same name.
+There are today two kinds of volumes:
+- `host` volumes that can expose a directory or a file from the host to the pod
+- `empty` volumes that initialize an empty storage to be accessed locally within the pod.
+
+Each volume can be selectively mounted into each application at differing mount points. Note that any volumes that are specified but do not have a matching mount point will be silently ignored.
+
+### Mounting Volumes
+
+For `host` volumes, the `--volume` flag allows you to specify each mount, its type, the location on the host, and whether the volume is read-only or not. In the following example, we make the host's `/srv/data` accessible to app1 on `/var/data`:
+
+```
+# rkt run --volume data,kind=host,source=/srv/data,readOnly=false example.com/app1
+```
+
+If you don’t intend to persist the data and you just want to have a volume shared between all the apps in the pod, you can use an `empty` volume:
```

Comment:
Whoa, how did you spot that? Is "look for unicode chars replaceable with ASCII ones" on your docs-review checklist?

Anyway, it's google docs' fault.